ui/helpers/utils/util.js

```diff
@@ -584,3 +584,30 @@ export const sanitizeString = (value) => {
   const regex = /\u202E/giu;
   return value.replace(regex, '\\u202E');
 };
+
+/**
+ * This method checks current provider type and returns its string representation
+ *
+ * @param {*} provider
+ * @param {*} t
+ * @returns
+ */
+
+export const getNetworkNameFromProviderType = (providerConfig, t) => {
```

Comment:
Not sure how large the providerConfig object is, but it might be better to pass only the name and nickname to this method as those are the only properties used. Plus drop `t` as per Jyoti's suggestion below.

```suggestion
export const getNetworkNameFromProviderType = (providerName, providerNickname) => {
```